### ISSUE TITLE
Helm universal argument broken. C-u 1 M-x broken.

### ISSUE THREAD

**Issue by loysollivier** [syl20bnr/spacemacs#13757]
#### Description :octocat:
C-u 1 M-x is broken.
The behavior has been broken by this commit:
c7ed2fc597e2 [helm] Provide fuzzy matching for Helm-M-x

#### Reproduction guide :beetle:
- Start Emacs
- Press the key combo: `C-1 M-x`

*Observed behaviour:* :eyes: :broken_heart:
Helm opens with prompt: `M-x`

*Expected behaviour:* :heart: :smile:
Helm should open with prompt: `1 M-x`

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3
- Spacemacs: 0.300.0
- Spacemacs branch: nil (rev. c7ed2fc597e2)
- Graphic display: t
- Distribution: spacemacs-base
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-tab-key-behavior 'complete auto-completion-enable-snippets-in-popup t :disabled-for org)
 better-defaults c-c++ cmake cscope csv deft emacs-lisp
 git gnus helm html javascript latex markdown notmuch org pdf python shell spacemacs-completion spacemacs-layouts spacemacs-editing spacemacs-editing-visual spacemacs-evil spacemacs-misc spacemacs-modeline spacemacs-navigation spacemacs-org spacemacs-project spacemacs-purpose spacemacs-visual
 (spell-checking :variables spell-checking-enable-by-default nil)
 (syntax-checking :variables syntax-checking-enable-tooltips nil syntax-checking-enable-by-default nil)
 version-control yaml)
```
- System configuration features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS GLIB NOTIFY LIBSELINUX GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM MODULES THREADS XWIDGETS LIBSYSTEMD LCMS2

#### Backtrace :paw_prints:
```
<<BACKTRACE IF RELEVANT>>
```

**Comment by loysollivier** [comment]
@smile13241324 FYI :)

**Comment by emacs18** [comment]
Following seems to fix the problem of dropping prefix argument:
```diff
diff --git a/layers/+completion/helm/funcs.el b/layers/+completion/helm/funcs.el
index 29c6cedae..7a4aaa1e6 100644
--- a/layers/+completion/helm/funcs.el
+++ b/layers/+completion/helm/funcs.el
@@ -644,4 +644,4 @@ to buffers)."
   (interactive)
   (let ((completion-styles completion-styles))
     (add-to-list 'completion-styles `,(if (version< emacs-version "27") 'helm-flex 'flex) t)
-    (command-execute 'helm-M-x)))
+    (call-interactively 'helm-M-x)))
```

However I don't know why `command-execute` was used (which I'm not familiar with) rather than `call-interactively`.

**Comment by smile13241324** [comment]
Ups, my mistake I have not thought about the universal argument and blindly introduced `command-execute` which could be replaced with `call-interactively`, command-execute is just the more flexible function which seems to lose the prefix argument somewhere on the way.

Many thanks for checking @emacs18 @loysollivier.
I'll fix this shortly.

**Comment by emacs18** [comment]
Hold on.  I think this change needs more testing.  I'm running into some issues using `call-interactively`.  I'll get back with details once I know more.

**Comment by emacs18** [comment]
Hmm.  I launched new emacs 27 and tested the changed code.  It seems to be working fine, i.e., `org-goto` is working normally where headline completion is done the usual helm way.  My original emacs stopped brining up the helm buffer with completions so I thought this problem may have been due to using `call-interactively`.

In short I do not see any problem using `call-interactively`, at least so far.
